style.css

```diff
@@ -0,0 +1,72 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root{
+  --white-color:#fff;

```

Comment:
We should not name colours like this. We are using variables for the purpose of reusability. Suppose if you want to change your particular colour of your background to blue that variable name that doesn't make sense. 

Ex- Your major color is gray then name it as primary color